includes/Story_Post_Type.php

```diff
@@ -567,6 +567,14 @@ public function get_editor_settings() {
 			admin_url( 'edit.php' )
 		);
 
+		$dashboard_settings_url = add_query_arg(
+			[
+				'post_type' => self::POST_TYPE_SLUG,
+				'page'      => 'stories-dashboard#/editor-settings',
+			],
+			admin_url( 'edit.php' )
+		);
+
```

Comment:
Added this link for user to link directly to the settings page. Not a php expert, so I imagine there's probably a better way to do this?

Reply:
Not a php expert here either but this looks good to me, it's not causing lint errors and it (most importantly) is working in the app.